minard/templates/burst_run_detail.html

```diff
@@ -0,0 +1,162 @@
+{% extends "layout.html" %}
+{% block title %}Burst Run Detail{% endblock %}
+{% block head %}
+  {{ super() }}
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/pca.css') }}" media="screen">
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        {{ subrun }}
+        {{ sub }}
+        <h2>Run {{ data["run"] }} details</h2>
+        <div class="row">
+          <div class="col-md-12">
```

Comment:
I think the row and column divs here are superfluous, i.e. you should use the outermost row and column to separate the table from the later panels.